distribution/zip/ballerina-tools/pom.xml

```diff
@@ -326,6 +326,12 @@
             <groupId>org.ballerinalang</groupId>
             <artifactId>language-server-stdio-launcher</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.ballerinalang</groupId>
+            <artifactId>composer-library</artifactId>
+            <type>zip</type>
+            <version>${project.version}</version>
```

Comment:
Shall we add the dependency to root pom's dependencyManagement section and have the version there, and inherit it from here?

Reply:
Modified the root pom as you suggested with [29e3658](https://github.com/ballerina-platform/ballerina-lang/pull/15412/commits/29e3658f38b7bdc43c5475544572e78c965f6e9d), and ended up with the following error.

``` 
'dependencies.dependency.version' for org.ballerinalang:composer-library:jar is missing. @ org.ballerinalang:ballerina-tools:[unknown-version], /home/nino/Desktop/ballerina/distribution/zip/ballerina-tools/pom.xml, line 329, column 21
```

Am I missing something? :)

Reply:
Fixed as per the offline discussion with @Manuri.